.flake8

```diff
@@ -1,2 +1,5 @@
 [flake8]
 max-line-length = 120
+per-file-ignores =
+    tests/test_stubs.py:F821
+    tests/test_typing.py:F821
```

Comment:
why should this lint rule be ignored in these files?

Reply:
I'd mentioned that in the summary:

> Note: I ran pipenv update -d to update the packages. This ended up upgrading pyflakes, which now errors on List[make_forward_ref('Foo')] saying "undefined name 'Foo'", probably because it's a literal non-class string within a container. So, I've temporarily silenced these errors in .flake8. Lmk if you have a good solution for that other than creating a bug report there.

Reply:
Ah, sorry for missing the summary! Yeah that's a lot of lint errors, I don't have a better solution.